geospaas/base_viewer/templates/base_viewer/origin_of_templates.html

```diff
@@ -0,0 +1,71 @@
+{% load static  %}
+{% load leaflet_tags %}
+<html lang="en">
+
+<head>
+	<title>Meta-data finder</title>
+	<meta charset="utf-8" />
+	{% leaflet_js plugins="ALL" %}
+	{% leaflet_css plugins="ALL" %}
+
+<!--
+	<style>
+
+		.leaflet-container {  /* all maps */
+			width:  100%;
+			height: 100%;
+		}
+	</style>-->
+	<style>
```

Comment:
Maybe move the CSS in a separate file in a `static` folder?

Reply:
Done